pkg/plugin/installer/installer_test.go

```diff
@@ -37,4 +41,8 @@ func TestIsRemoteHTTPArchive(t *testing.T) {
 	if isRemoteHTTPArchive(source + "-not-an-extension") {
 		t.Error("Expected media type match to fail")
 	}
+
+	if !isRemoteHTTPArchive(binarySource) {
+		t.Errorf("Expected %q to be a valid archive URL", source)
```

Comment:
I was trying to spot why @robertsirc got an error, and I noticed:

```suggestion
		t.Errorf("Expected %q to be a valid archive URL", binarySource)
```